sfge/src/main/java/com/salesforce/graph/ops/MethodUtil.java

```diff
@@ -613,7 +619,6 @@ public static List<ApexPath> getPaths(
                                 .orElse(null);
             } else if (vertex instanceof ArrayLoadExpressionVertex) {
                 // Intentionally left blank
-                // TODO: Why?
```

Comment:
Can we remove this `else if` block, or does keeping it somehow preserve clarity?